src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -6092,6 +6092,75 @@
     public static final int PERMITS_CLAUSE =
         JavaLanguageLexer.PERMITS_CLAUSE;
 
+    /**
+     * A pattern definition, excluding simple type pattern (pattern variable)
+     * definition such as {@code if (o instanceof Integer i){}}. Pattern definitions
+     * appear as operands of statements and expressions.
+     *
+     * <p>For example:</p>
+     * <pre>
+     * switch(o) {
+     *     case String s &amp;&amp; s.length() &gt; 4: // guarded pattern, `PATTERN_DEF`
```

Comment:
please update example for "intanceof" 
https://checkstyle.sourceforge.io/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html
with extra example for PATTERN_DEF

Reply:
Done.

Reply:
Please make it "same" as existing example to see a difference. See as you did in pattern_def

Reply:
Good idea, done.